README.md

```diff
@@ -41,7 +41,6 @@ Stubs for the following libraries now exist in typeshed or the libraries themsel
 - filelock
 - freezegun
 - jmespath
-- matplotlib
```

Comment:
Maybe this should still be here? With a potential explanation as to why we still have stubs for it?

Reply:
Ok, how does this look?